### PR TITLE
[1LP][WIP] Add test_reconfigure_instance

### PR DESCRIPTION
**Pull request by jadhaj**
Purpose or Intent
=================
This test depends on the following issue:
https://github.com/RedHatQE/widgetastic.core/issues/113

{{ pytest: cfme/tests/openstack/cloud/test_flavors.py::test_reconfigure_instance }}

### CODE REVIEW

File: cfme/tests/openstack/cloud/test_flavors.py [hunk 1]
```diff
@@ -123,3 +123,11 @@ def cleanup():
 
     wait_for(lambda: not flavor.exists, delay=5, timeout=600, fail_func=flavor.refresh,
              message='Wait for flavor to appear')
+
+
+@pytest.mark.ignore_stream('5.9')
```
Comment (sshveta) [inline cfme/tests/openstack/cloud/test_flavors.py]: Can you please comment why we are ignoring this for 5.9 ?

Comment (jadhaj) [reply cfme/tests/openstack/cloud/test_flavors.py]: because zero flavor disk is not supported in 5.9 yet

**Comment by mshriver** [comment]
Depends on wt.core issue that is unresolved, WIP for now.

**Comment by dajoRH** [comment]
Lint report for commit 930b1a4c96919c7bd892cf6c5efd506ea530bb9b:

`cfme/tests/openstack/cloud/test_flavors.py`:
- :bangbang: Line 129:43: **E231** *missing whitespace after ','*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 589e7f012ed53f4a761c60ea32e4919c10a2af00:

`cfme/tests/openstack/cloud/test_flavors.py`:
- :heavy_exclamation_mark: Line 297:1: **W391** *blank line at end of file*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Pull request closed without merging**